src/Microsoft.Health.Fhir.Liquid.Converter/Models/DateTimeObject.cs

```diff
@@ -0,0 +1,137 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.Models
+{
+    public class DateTimeObject
+    {
+        public DateTimeOffset DateValue { get; set; }
+
+        public bool HasTimeZone { get; set; } = true;
+
+        public bool HasTime { get; set; } = true;
+
+        public bool HasMilliSecond { get; set; } = true;
```

Comment:
do we need so many fields? how about change it to Has(date,time,timezone), which is much easier to use?